docs/reference/src/main/asciidoc/injection.asciidoc

```diff
@@ -551,6 +558,12 @@ Annotation qualifier = synchronously ?
 PaymentProcessor p = anyPaymentProcessor.select(qualifier).get().process(payment);
 ----------------------------------------------------------------------------------
 
+[NOTE]
```

Comment:
I would move this note to `4.4. The built-in qualifiers @Default and @Any` chapter